src/main/steps/tasklistresponse/proceedings/navigationController.ts

```diff
@@ -103,6 +82,49 @@ class OtherProceedingsNavigationController {
     }
     return nextUrl;
   }
-}
 
+  private getNextUrlOtherProceedingDetails(caseData) {
```

Comment:
has this been unit tested? I don't think it's going to work. Please check the flow.

Reply:
The correct flow is corrected now. The flow is tested through getNextUrl function and now it is 94.64 coverage of lines of code.